src/components/CreateFileForm/CreateFileForm.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from "react";
+import {
+  Box,
+  Button,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+  TextField,
+} from "@mui/material";
+import { useCreateFileMutation } from "../../services/task1";
+import { FileFormBody } from "../../types";
+
+export const CreateFileForm = () => {
+  const [fileForm, setFileForm] = useState<FileFormBody>({
+    filename: "",
+    content: "",
+    type: "",
+    password: "",
+  });
+
+  const [createFile, { isLoading, isError, error, isSuccess }] =
+    useCreateFileMutation();
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target;
+    setFileForm({ ...fileForm, [name]: value });
+  };
+
+  const handleSelectChange = (e: SelectChangeEvent) => {
+    const { name, value } = e.target;
+    setFileForm({ ...fileForm, [name]: value });
+  };
+
+  const handleFileCreate = () => {
+    createFile(fileForm);
+  };
+
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+        marginTop: 5,
+      }}
+    >
+      <TextField
+        sx={{ width: "200px", marginTop: "10px", input: { color: "#fff" } }}
```

Comment:
you can use mui grid component to perform such styling:
https://mui.com/material-ui/react-grid/
```
<Grid container spacing={1} direction="column">
  <Grid item>
    <TextField/>
  </Grid>
  ....
```
As for re-usable styles, you can crete styled component, e.g.
`StyledTextField = styled(TextField)({})`